docs/user-guide/dev-environment.md

```diff
@@ -660,6 +660,10 @@ ARCHER2 currently has the following HPE Cray Programming Environment releases av
 - **21.04: Current default**
 - 21.09: available via `cpe/21.09` module
 
+You can find information, notes, and lists of changes for current and upcoming ARCHER2 
+HPE Cray programming environments in the HPE Cray Programming Environment GitHub
+repository found [here](https://github.com/PE-Cray).
```

Comment:
Can you change the hyperlink to contain descriptive text rather than "here"? This is needed for accessibility reasons. For example: 

You can find information, notes, and lists of changes for current and upcoming ARCHER2 
HPE Cray programming environments in [the HPE Cray Programming Environment GitHub
repository](https://github.com/PE-Cray).

Reply:
Good spot -- I've fixed it now.